airflow/providers/google/cloud/transfers/azure_blob_to_gcs.py

```diff
@@ -122,3 +122,20 @@ def execute(self, context: Context) -> str:
                 self.bucket_name,
             )
         return f"gs://{self.bucket_name}/{self.object_name}"
+
+    def get_openlineage_facets_on_start(self):
+        from openlineage.client.run import Dataset
+        from airflow.providers.openlineage.extractors import OperatorLineage
+
+        wasb_hook = WasbHook(wasb_conn_id=self.wasb_conn_id)
+        account_name = wasb_hook.get_conn().account_name
```

Comment:
Should we add the test (probably in `tests/providers/microsoft/azure/hooks/test_wasb.py`) that checks if the account_name is properly extracted?

Reply:
added 👍🏻